src/mrvi/_model.py

```diff
@@ -61,7 +62,7 @@ class MrVI(JaxTrainingMixin, BaseModelClass):
         AnnData object that has been registered via ``setup_anndata``.
     n_latent
         Dimensionality of the latent space.
-    n_latent_donor
+    n_latent_sample
```

Comment:
This arg is being passed into `MrVAE` but it doesn't look like `MrVAE` has this argument defined.

Reply:
ah good catch, its an old one